v2/emailpassword/serverless/with-netlify/frontend.mdx

```diff
@@ -83,11 +83,18 @@ SuperTokens.init({
 >
 
 ```tsx
-import supertokens from "supertokens-website-script";
+import supertokens from "supertokens-web-js-script";
```

Comment:
Need to also change <TabItem value="npm"> one

Reply:
Done in a separate PR